ui/lib/kv/addon/routes/secret/metadata.js

```diff
@@ -7,31 +7,63 @@ import Route from '@ember/routing/route';
 import { service } from '@ember/service';
 
 export default class KvSecretMetadataRoute extends Route {
-  @service store;
+  @service capabilities;
```

Comment:
calling this service in the route file has been a newer pattern I've adopted during this subkey/patch work. Tests were failing because the model capability checks were not consistently returning what we expected in the test suite.